src/business_logic/transaction/objects/internal_declare.rs

```diff
@@ -849,7 +849,7 @@ mod tests {
 
         assert_matches!(
             internal_declare.execute(&mut state, &StarknetGeneralConfig::default()),
-            Err(TransactionError::ResourcesError { .. })
```

Comment:
Previously it failed because `cairo_resource_usage` didn't contain all the resources we now have defined in the config. (Was `{"pedersen_builtin": 15, "range_check_builtin": 57, "l1_gas_usage": 0}`.)

The actual fee is zero BTW, because gas_price is 0 in the config.

Now it fails because of a fee transfer failure while executing the transfer because the fee token contract is not actually deployed.